### ISSUE TITLE
[Preview] react-devtools console warning: Module not found: Can't resolve 'holderjs'

### ISSUE THREAD

**Issue by mbrookes** [jquense/jarle#7]
## Current Behavior 😯

```
../node_modules/jarle/esm/Preview.js
Module not found: Can't resolve 'holderjs' in '/Users/mbrookes/Projects/material-ui/node_modules/jarle/esm'
```

_ | _ | _
-- | -- | --
overrideMethod | @ | VM114144 react_devtools_backend.js:2430
printWarnings | @ | hot-dev-client.js?647e:126
handleWarnings | @ | hot-dev-client.js?647e:131
processMessage | @ | hot-dev-client.js?647e:213
eval | @ | hot-dev-client.js?647e:54
eval | @ | eventsource.js?9b42:39
handleMessage | @ | eventsource.js?9b42:37

## Expected Behavior 🤔

No warning

## Steps to Reproduce 🕹

1. Use Jarle in Chrome with react-devtools
2. Check the console

## Context 🔦

Testing Jarle for use in React component documentation, where developers checking the console will not expect to see errors.

**Comment by jquense** [comment]
Yeah this is do to an optional require that the editor makes that webpack warns about.ineed to see if there is a good way to hint that it's fine it's not found

**Comment by mbrookes** [comment]
Yeah, I'd traced it that far. I had hoped there might be a way to suppress the warning altogether...

**Comment by Adam-Collier** [comment]
This warning also shows in Vite:
<img width="504" alt="image" src="https://user-images.githubusercontent.com/20065745/133077365-83fa0b42-68db-4937-912b-7fded2840126.png">

Does `holderjs` not need adding to the dependencies?
